build.sbt

```diff
@@ -256,8 +264,9 @@ lazy val tools =
     .settings(
       libraryDependencies ++= Seq(
         scalacheckDep,
-        scalatestDep
-      ),
+        scalatestDep,
+        collectionsCompatLib
```

Comment:
There are only 3 uses for collections compat, on which I have commented. I think we should try to get rid of them so that we don't need collections-compat here. That would remove all the remaining uses of scala-collections-compat.

Reply:
Yeah, in meantime I've created commit removing them and pararell collections library completely. Only 2 remaining usages of `JavaConverters` are in tools which are always compiled with 2.12